src/config.ts

```diff
@@ -328,6 +330,13 @@ export function getConfig(options?: { reload?: boolean }): StorageConfigType {
       getOptionalConfigFromEnv('RATE_LIMITER_REDIS_COMMAND_TIMEOUT') || '2',
       10
     ),
+  } as StorageConfigType
+
+  if (!config.isMultitenant && !config.serviceKey) {
```

Comment:
won't this generate different servicekeys each time the server is restarted or if this is called?

Reply:
As discussed, it's fine if we generate the service key at boot time for single tenants